bucket/tableplus.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "110",
+    "description": "Relational database explorer",
+    "homepage": "https://tableplus.com",
+    "license": "Proprietary",
+    "url": "https://s3.amazonaws.com/tableplus-win-builds/110/TablePlusSetup.exe",
+    "hash": "be696fb0c6f1f39c51cae409e02a364ebcda0172c21cb42c941430f8a9eedce5",
+    "innosetup": true,
+    "architecture" :{
+        "32bit": {
+            "pre_install": "Get-ChildItem \"$dir\" '*,1*' | ForEach-Object { Rename-Item $_.Fullname ($_.Name -Replace ',2') }"
```

Comment:
- Did you check if Recurse is needed? 
- rename-item do not nebes foreach. `| Rename-item {$_.FullName } {replace +$_.ext}`

Reply:
> * Did you check if Recurse is needed?

I did checked the files in the *install script*. All the files with `,1` suffix is under `{app}` directory, so `-Recurse` is not needed.